sonar-plugin-api/src/main/java/org/sonar/api/platform/Server.java

```diff
@@ -101,8 +101,9 @@
   public abstract String getPublicRootUrl();
 
   /**
-   * The dev mode is enabled when the property {@code sonar.web.dev} is {@code true}.
-   *
+   * Before version 6.6, the dev mode is enabled when the property {@code sonar.web.dev} is {@code true}.
+   * Since 6.6, {@code false} is always returned.
+   * @deprecated in 6.6
    * @since 5.4
    */
   public abstract boolean isDev();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @Deprecated annotation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)